### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by BenMorel**

### CODE REVIEW

File: reference/bc/functions/bcscale.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
        la précision actuelle ; quand elle est utilisé pour définir une nouvelle
```
Comment (villfa) [inline reference/bc/functions/bcscale.xml:75]: It's the opportunity to fix this too:
```suggestion
       la précision actuelle ; quand elle est utilisée pour définir une nouvelle
```

Comment (BenMorel) [reply reference/bc/functions/bcscale.xml]: Good catch. Fixed!

File: reference/bc/functions/bcscale.xml [hunk 2]
```diff
@@ -50,7 +50,7 @@
    Sinon la précision actuel est retorunée.
```
Comment (orklah) [inline reference/bc/functions/bcscale.xml:50]: Précision actuelle?

Comment (orklah) [reply reference/bc/functions/bcscale.xml:50]: `retournée` too 😄

Comment (BenMorel) [reply reference/bc/functions/bcscale.xml]: OMG. So many typos. Fixed!

Comment (Girgias) [reply reference/bc/functions/bcscale.xml]: Typos are recurrent due to how the processed worked >_>

**Review by villfa — APPROVED** [review]

**Pull request closed without merging**